app/javascript/components/supplemental_application/actions.js

```diff
@@ -18,3 +18,21 @@ export const getAMIAction = async ({chartType, chartYear}) => {
   }
   return response['ami']
 }
+
```

Comment:
@fddayan has mentioned that we likely do not really need this actions file going forward. if people would like to remove these functions and place their content directly in the place where they are called in the supplemental application page, i would be fine with that. i'm also fine with leaving this this way until we have a more solid idea of how we want to organize our React requests to Rails.